twindle-cli/twitter/utils/process-tweet-entities.js

```diff
@@ -182,6 +182,7 @@ function renderMentionsHashtags({ text = "", mentions = [], hashtags = [] }) {
 /**
  * Fix user description from multiple tweets combined obj. DO NOT COMPOSE IN THE RENDERRICHTWEETS FUNCTION
  */
+
```

Comment:
JSDoc won't work if there's a space in between. PLease remove the blank line

Reply:
Then why there is other blank lines in file 😅

Reply:
Please remove this line 185 here.